softdev.bib

```diff
@@ -2445,3 +2445,37 @@ @inproceedings{oliveira14howmuch
     year = 2014,
     month = Feb
 }
+
+@article{mcilroy19spectre,
+    author = "Ross McIlroy and Jaroslav Sevc{\'{\i}}k and Tobias Tebbi and Ben L. Titzer and Toon Verwaest",
```

Comment:
Did you check this non-ascii glyph renders ok?

Reply:
It does render OK, but I've used a simplification that also renders the same way (so I think is better) -- see what you think.

Reply:
Much more readable :)